packages/firestore/lite/test/integration.test.ts

```diff
@@ -832,4 +836,134 @@ describe('Query', () => {
       }
     );
   });
+
+  it('supports collection groups', () => {
+    return withTestCollection(async collRef => {
+      const collectionGroupId = `${collRef.id}group`;
+
+      const fooDoc = doc(
+        collRef.firestore,
+        `${collRef.id}/foo/${collectionGroupId}/doc1`
+      );
+      const barDoc = doc(
+        collRef.firestore,
+        `${collRef.id}/bar/baz/boo/${collectionGroupId}/doc2`
+      );
+      await setDoc(fooDoc, { foo: 1 });
+      await setDoc(barDoc, { bar: 1 });
+
+      const query = collectionGroup(collRef.firestore, collectionGroupId);
+      const result = await getQuery(query);
+
+      verifyResults(result, { bar: 1 }, { foo: 1 });
+    });
+  });
+
+  it('validates collection groups', () => {
+    return withTestDb(firestore => {
+      expect(() => collectionGroup(firestore, '')).to.throw(
+        'Function collectionGroup() requires its first argument to be of type non-empty string, but it was: ""'
+      );
+      expect(() => collectionGroup(firestore, '/')).to.throw(
+        "Invalid collection ID '/' passed to function collectionGroup(). Collection IDs must not contain '/'."
+      );
+    });
+  });
+});
+
+describe('equality', () => {
+  it('for collection references', () => {
```

Comment:
consider using `context()` blocks to setup test context, and then use `it()` blocks for expectations

```
describe('equality', () => {
  context('for collection references', () => {
   it("returns true when they are equal", () => {} );
   it("returns false when they are not equal", () => {});
  });
});
```

Reply:
I have never seen this context before. I will add a TODO. 

FYI I am planning to address all TODOs before releasing, but I want to avoid huge merge conflicts.